silverback/_cli.py

```diff
@@ -40,7 +44,23 @@ def _network_callback(ctx, param, val):
     return val
 
 
-@cli.command()
+async def run_worker(broker: AsyncBroker, worker_count=2, shutdown_timeout=90):
+    try:
+        with ThreadPoolExecutor(max_workers=worker_count) as pool:
+            receiver = Receiver(
+                broker=broker,
+                executor=pool,
+                validate_params=True,
+                max_async_tasks=1,
+                max_prefetch=0,
+            )
+            broker.is_worker_process = True
+            await receiver.listen()
+    finally:
+        await shutdown_broker(broker, shutdown_timeout)
+
+
+@cli.command(help="Run Silverback application")
```

Comment:
```suggestion
@cli.command(help="Run Silverback application client")
```

Reply:
I wonder if we should leave out any talk of distributed installs, except for a part of the doc explaining that configuration?  Adding "client" here seems like it could also be confusing in single-instance memory broker setups.

Reply:
Sorta why I only want the SDK to support sqlite out of the box, so you can see the persistence in action for the purposes of local dev, but not what the full platform will offer you (when it's available)

Reply:
I updated the docs and added a section on distributed execution.  Cleaned up some other references to distributed execution that weren't useful.

But there's not a lot of room to make this help text informative.

```
$ silverback --help
Usage: silverback [OPTIONS] COMMAND [ARGS]...

  Work with Silverback applications in local context (using Ape).

Options:
  --help  Show this message and exit.

Commands:
  run     Run Silverback application (distributed client)
  worker  Run Silverback application task workers
```

Suggestions?

Reply:
deja vu.  sure.